### DISCUSSION TITLE
Add the possibility for the user to change the name of the Solution when importing the Template.

### DISCUSSION THREAD

**Discussion by barcellosrenan** [microsoft/PowerPlatform-Catalog#30] [Ideas]
It would be interesting to have a new configuration step during catalog import where the user could define what name they want for the solution before importing. 

![image](https://github.com/user-attachments/assets/a81f155f-3f26-4b19-8a16-3843afeb414b)

**Comment by dfreitasnunes** [comment]
I agree with this, because when using the template for the second time it seems it will overwrite because the name is the same.

**Comment by larhem** [comment]
Both displayname and internal name would be great if you could give name to on import
